package.json

```diff
@@ -61,7 +61,7 @@
     "pattern library"
   ],
   "dependencies": {
-    "glamor": "^2.20.39",
+    "glamor": "^2.20.0",
```

Comment:
why the downgrade?

Reply:
just to keep the versions in the same range of everything else. i figured it would resolve to the same major version but lerna likes these version in the package.json files to be exact